front/src/modules/simulationResult/components/SpeedSpaceChart/consts.ts

```diff
@@ -24,3 +24,18 @@ export const electricalProfileColorsWithoutProfile = {
   '15000V': '#009AA6',
   '3000V': '#1FBE00',
 };
+
```

Comment:
Should we have a comment with an url to link the sketch version corresponding to these colors mapping ?

Reply:
why?

Reply:
Probably not needed